jiratools/jira.config.example

```diff
@@ -6,9 +6,9 @@ PASSWORD=YOUR_PASSWORD
 DEFAULT_ASSIGNEE=%(USERNAME)s
 TEST_PROJECT=YOUR_JIRA_PROJECT
 DEFAULT_ISSUE_TYPE=Story
-DEFAULT_DESCRIPTION=Do necessary QE testing around the linked Dev JIRA
+DEFAULT_DESCRIPTION=Linking a new story to {dev_jira_id} using jiratools cli.
```

Comment:
```suggestion
DEFAULT_DESCRIPTION=Do necessary work related to linked JIRA.
```

Reply:
Note that, as it currently stands, no formatting values are passed to the `description` field -- it's not used as a template, just a string. To fill in values, you'd need to update the code behidn it as well.